src/lib/wallets/lobstr/__test__/lobstr.test.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { expect } from '@jest/globals';
+import { signTransaction } from '@lobstrco/signer-extension-api';
+import * as StellarSdk from 'stellar-sdk';
```

Comment:
Can we import just what we need instead of the whole library?

Reply:
done